utils.py

```diff
@@ -3,11 +3,20 @@
 import numpy as np
 import os
 
-def load_entityset(data_dir):
-    order_products = pd.read_csv(os.path.join(data_dir, "order_products__prior.csv"))
-    orders = pd.read_csv(os.path.join(data_dir, "orders.csv"))
-    departments = pd.read_csv(os.path.join(data_dir, "departments.csv"))
-    products = pd.read_csv(os.path.join(data_dir, "products.csv"))
+def load_order_products(data_dir, nrows=None):
```

Comment:
does this method get used?

Reply:
No, this method doesn't get used. The `make_labels` and `dask_make_labels` also don't get used. I can remove these methods.